src/linter/block_linter.go

```diff
@@ -598,11 +598,15 @@ func (b *blockLinter) checkArray(arr *ir.ArrayExpr) {
 	haveImplicitKeys := false
 	keys := make(map[string]ir.Node, len(items))
 
-	for _, item := range items {
+	multiline := arr.Position.EndLine-arr.Position.StartLine != 0
```

Comment:
First of all, you could write that as
```go
multiline := arr.Position.EndLine != arr.Position.StartLine
```

Second of all, I would suggest to not try to be clever and just write it like this:

```go
multiline := false
if arr.Position.EndLine != arr.Position.StartLine {
multiline = true
}
```

The last version is much easier to understand.

Reply:
Done.